### ISSUE TITLE
kernel obj format - is it still valid?

### ISSUE THREAD

**Issue by DanielMazurkiewicz** [gpujs/gpu.js#417]
Is it still a valid format?
https://github.com/gpujs/gpu.js/blob/develop/src/notes/kernel-obj-format.js

Aldo, there is no definition of "exportKernelObj" method in entire project.

**Comment by robertleeplummerjr** [comment]
No, I'm removing it for v2.  I'll have the format for you today.
